src/lib/components/processing/AddOns.svelte

```diff
@@ -0,0 +1,152 @@
+<!-- @component
+Fetch and display status of all running add-ons.
+This component should update on a timer.
+-->
+<script context="module" lang="ts">
+  import type { Run } from "@/addons/types";
+  import { writable, type Writable } from "svelte/store";
+
+  export const running: Writable<Run[]> = writable([]);
+</script>
+
+<script lang="ts">
+  import { afterUpdate, onMount, onDestroy } from "svelte";
+  import { flip } from "svelte/animate";
+  import { _ } from "svelte-i18n";
+  import { Plug16, Thumbsdown16, Thumbsup16, XCircle16 } from "svelte-octicons";
+
+  import SidebarGroup from "../sidebar/SidebarGroup.svelte";
+  import SidebarItem from "../sidebar/SidebarItem.svelte";
+
+  import Button from "../common/Button.svelte";
+  import Flex from "../common/Flex.svelte";
+  import Process from "./Process.svelte";
+
+  import { POLL_INTERVAL } from "@/config/config";
+  import { history, dismiss, cancel, rate } from "$lib/api/addons";
+  import { getCsrfToken } from "$lib/utils/api";
+
+  let timeout: string | number | NodeJS.Timeout;
+
+  onMount(async () => {
+    if ($running.length === 0) {
+      await load();
+    }
+  });
+
+  afterUpdate(() => {
+    if ($running.length > 0) {
+      timeout = setTimeout(load, POLL_INTERVAL);
+    }
+  });
```

Comment:
This won't check for dispatched addons if there are not any currently running. How do we make sure that in a clean session, dispatching an add-on populates the `$running` store?